client/src/components/create-page/CreateProfile.js

```diff
@@ -94,13 +94,23 @@ export default function CreateProfile() {
 
   function handleBioChange(e) {
     const newBio = e.target.value;
-
     setBio(newBio);
+    const text = e.nativeEvent.target.textLength;
     setCounter(counter - 1);
-    if (e.target.value === "") {
+
+    if (e.nativeEvent.inputType === "insertText") {
+      setCounter(counter - 1);
+    } else if (e.nativeEvent.target.textLength === 0) {
       setCounter(125);
-    } else if (e.nativeEvent.data === null) {
+    } else if (e.nativeEvent.inputType === "deleteContentBackward") {
       setCounter(counter + 1);
+    } else if (e.nativeEvent.inputType === "insertLineBreak") {
+      setCounter(counter - 1);
+    } else if (
+      e.nativeEvent.inputType === "deleteSoftLineBackward" ||
+      e.nativeEvent.inputType === "deleteWordBackward"
+    ) {
+      setCounter(125 - text);
     }
```

Comment:
I recommend having this either be part of the "Bio" component or put it in a separate function